### DISCUSSION TITLE
LEAK - A good view on Imminent Demise!

### DISCUSSION THREAD

**Discussion by Pico8430** [Pico8430/Indie-Cross-PE-Port#15] [Announcements]
# Oh boy, here it comes!

https://user-images.githubusercontent.com/91833725/168177152-1d65c886-f15f-4284-b0db-e4e2c04acd37.mp4

#

**Comment by Superiron12321** [comment]
One thing that many psych engine ports of indie cross miss, is the brightness change of the background. You should try adding that.

**Reply by Pico8430** [reply]
Got it, thanks.
